test/deps/executables_test.dart

```diff
@@ -190,56 +190,4 @@ main() {
     test("are listed if --dev flag is set", _testAllDepsOutput('foo:bar'));
     test("are skipped if --no-dev flag is set", _testNonDevDepsOutput('\n'));
   });
-
-  group("lists dependencies overrides", () {
```

Comment:
I'll try it by I think it will not pass because of [this](https://github.com/dart-lang/pub/pull/1680/commits/dc00b79896666c8d9accabd57395411692242e36)

Actually, it will pass in case of `--dev` flag, but it will fail with `--no-dev`

Thats was my question [here](https://github.com/dart-lang/pub/pull/1680#discussion_r131791096)